src/app/story/storyrow/storyrow.component.css

```diff
@@ -50,11 +43,70 @@
     margin: 0;
 }
 
+.node-uid-column__display-name {
+    font-weight: bold;
+}
+
+.node-uid-column__timestamp {
+    font-size: 1.1rem;
+}
+
+.clock-icon {
+    /* We want it to be on top of everything except the sidebar */
+    z-index: 97;
+    margin-left: 4.1rem;
+    bottom: 1.2rem;
+    position: absolute;
+    width: 12%
+}
+
+.wait-time--active {
+    top: 9.9rem !important;
+}
+
+.wait-time {
+    position: absolute;
+    right: -2rem;
+    top: 9.5rem;
+}
+
+.lengthened {
+    display: none;
+}
+
+.shortened {
+    display: initial;
+}
+
+.timezone {
```

Comment:
Could you rename this to `node-uid-column__timestamp__timezone` and move it to the "Modifiers" section of the file?